docs/_Index.md

```diff
@@ -0,0 +1,16 @@
+---
+github: https://github.com/JoinColony/colonyNetwork
```

Comment:
Please remove this file and replace it with a json config similar to https://github.com/JoinColony/colonyJS/blob/docs/improve-doc-config/docs/doc.config.json